packages/openapi-to-graphql/src/oas_3_tools.ts

```diff
@@ -1026,64 +1026,78 @@ export function getSecurityRequirements(
   return results
 }
 
+export enum CaseStyle {
+  PascalCase, // Used for type names
+  camelCase, // Used for (input) object field names
+  ALL_CAPS // Used for enum values
+}
+
 /**
  * First sanitizes given string and then also camel-cases it.
  */
-export function sanitize(
-  str: string,
-  lowercaseFirstChar: boolean = true
-): string {
+export function sanitize(str: string, caseStyle: CaseStyle): string {
```

Comment:
sanitize() now takes in a case. Before we were changing the case everywhere it was not very clear.

Reply:
I like that.